license.md

```diff
@@ -1,23 +1,7 @@
-Copyright (c) 2015, SilverStripe
-All rights reserved.
+Copyright (c) 2015 Christopher Pitt
```

Comment:
@hafriedlander do we apply this same reasoning to blog or any of the other similar modules in the SilverStripe organisation?